src/main/java/ng/abdlquadri/eventbus/EventBus.java

```diff
@@ -232,17 +244,21 @@ public static void close() {
       closeFuture.addListener(new ChannelFutureListener() {
         public void operationComplete(ChannelFuture future) throws Exception {
           if (future.isSuccess() && future.isDone()){
-            log.log(Level.SEVERE, "Channel Successful Closed");
+           // silent exit is better
```

Comment:
@nielsbaloe 

> this essentially means we are not reacting to the closeFuture?

Both true and false have been commented out. so nothing happens when close is called.